.github/workflows/windows.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           ${{ runner.os }}-maven-
     - name: Build with Maven
       run: |
-        mvn -B clean install -Dall -T2 --file pom.xml
+        mvn -B clean install -Dall -T2 --file pom.xml -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 --file src/pom.xml -Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss,SSS
```

Comment:
weird build failure on windows now... maybe because the `--file pom.xml` is specified twice